tests/tinytest.R

```diff
@@ -0,0 +1,5 @@
+
+if ( requireNamespace("tinytest", quietly=TRUE) ){
+  tinytest::test_package("inline")
```

Comment:
Have a look at what I do in e.g. `nanotime`, `RcppSpdlog`, ...  We probably want `R_TESTS` but who knows maybe `tinytest` now sets it too.   @markvanderloo ?   I also call `set.seed()` which never hurts.

Reply:
I had a look at nanotime. Regarding `R_TESTS`, this is now taken care of in tinytest@0ab06593a51029a44a644c46f890fd424302077c Chances are this would have fixed my trouble with the simple test script. With my next commit, we are only running the tests if the tinytest version is >= 1.1.0 which has this commit. Regarding the seed, I usually set it in the file where I am using it to make it clear that random numbers are involved. But it will still not hurt in the caller script, so I added that too.

Reply:
Awesomeness!  I was unaware -- I will carry that over to other `tinytest` users.  Thanks for digging that up.

And agreed on the seed setter. I usually set it in the test script too.  But no harm done.